### ISSUE TITLE
Commands not being recognized (persisting issue)

### ISSUE THREAD

**Issue by l1n00x** [texstudio-org/texstudio#3199]
Using txs 4.6.0b2, the issue still persists. I used my previous example which works if it is copied into the txs editor. If txs is closed and reopened however, commands are not recognized within 01.tex still.

 **main.tex**
 
 ```
 \documentclass{article}
 \usepackage[utf8]{inputenc}
 \usepackage[
 	backend=biber,
 ]{biblatex}
 
 \begin{document}
 
 \include{01.tex}
 \parencite{bibid}
 
 \end{document}
 ```
 
 **01.tex**
 
 ```
 \parencite{bibid}
 ```

_Originally posted by @l1n00x in https://github.com/texstudio-org/texstudio/issues/3180#issuecomment-1623679444_

**Comment by l1n00x** [comment]
It seems to be connected to opening another file.

How to reproduce:

1. Open a project by double clicking the tex file (txs is closed)
2. Close all tabs with open files, then close txs
3. Open another project by double clicking the tex file (check if the commands within the included files are recognized)
4. Repeat 2.
5. Repeat 1.

It may take several tries. Maybe you could open up additional projects if the above steps did not yield any results. If a project is reopened without any other project opened in between, the issue does not occur. I have produced a little video of the issue where I did not strictly follow the obove procedure, but it gets the job done ;)

Second project used:
**main2.tex**
```
\documentclass{beamer}
\setbeamertemplate{caption}[numbered]
\usetheme{CambridgeUS}

\usepackage[
	style = apa
]{biblatex}

\begin{document}

\include{01}

\end{document}
```

**01.tex**
```
\parencite{bibid}
```

https://github.com/texstudio-org/texstudio/assets/137958291/6c393536-677f-4f46-9d00-c28f591e8928

**Comment by l1n00x** [comment]
Seemingly fixed in 4.6.3beta1
